docs/src/pages/lab/layout/FixedContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/lab/Container';
+
+function FixedContainer() {
+  return (
+    <React.Fragment>
+      <CssBaseline />
```

Comment:
The idea is to only have on container in your app? Otherwise I would remove this and stick with advising devs to use this in their root component.

Reply:
No, you can have as many containers you want, one for the content, one for the footer, one for the hero, etc.

Reply:
Then I would suggest we remove this here. Creates a lot of redundant component instances.

Reply:
The demo is in an iframe, we would have to remove the body margin. I want to showcase the best way to start a top-level layout.

Reply:
Oh nevermind. I thought I was looking at a component from the core. I guess this is fine in demos.